### DISCUSSION TITLE
新年新希望

### DISCUSSION THREAD

**Discussion by seastar0687** [1101-GS4719-C/Python-Community-GS4719-C#3696] [General]
大家有甚麼新年新希望嗎
新年是一個新的開始，也是告別以前不好的事物的好時機
這時許下新的期許，有助於在新的一年更有目標跟動力喔
我自己想要明年可以完成我的研究並丟PAPER到研討會上
希望大家一起在今年的最後一天向明年許願吧

**Comment by kai1660** [comment]
我希望我的Python 程式設計可以滿分，並且學到許多程式相關的知識，幫助我日後的學習，我希望我的貓咪健健康康不要一直生病，畢竟生病很花錢的，我也希望我不要生病，免疫力可以高一點。
